frontend/src/pages/Profile.tsx

```diff
@@ -1,113 +1,115 @@
-import {useNavigate} from "react-router-dom";
-import {useState} from "react";
+import axios from "axios";
+import { jwtDecode } from "jwt-decode";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
 import validator from "validator";
-import {jwtDecode} from "jwt-decode";
 import { PasswordStrengthValidator } from "../components/PasswordStrengthValidator";
-import axios from "axios";
 
-type User = { firstName: string; lastName: string; sub: string }
+type User = { firstName: string; lastName: string; sub: string };
 
 const decodeToken = (): User | null => {
-    const token = localStorage.getItem('token');
-    if (token) {
-        const decodeToken = jwtDecode<User>(token);
-        console.log(decodeToken);
-        return decodeToken;
-    }
-    return null;
+  const token = localStorage.getItem("token");
+  if (token) {
+    const decodeToken = jwtDecode<User>(token);
+    return decodeToken;
+  }
+  return null;
 };
 
 export function Profile() {
-    const userFromToken = decodeToken();
-    const [password, setPassword] = useState("");
-    const [user, setUser] = useState<User | null>(userFromToken);
-    const [errMessage, setErrMessage] = useState<string | null>(null);
-    const [message, setMessage] = useState("");
-    const navigate = useNavigate();
+  const userFromToken = decodeToken();
+  const [password, setPassword] = useState("");
+  const [user, setUser] = useState<User | null>(userFromToken);
+  const [errMessage, setErrMessage] = useState<string | null>(null);
+  const [message, setMessage] = useState("");
+  const navigate = useNavigate();
 
-    const handleUserChange = (e: React.ChangeEvent<HTMLInputElement>) => {
-        const {name, value} = e.target;
-        setUser({...user!, [name]: value})
+  const handleUserChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = e.target;
+    if (user != null) {
```

Comment:
Do we need this If statement? Normally Profile page is invisible until user is logged in.

Reply:
The linter was unhappy with the other version.